guides/common/modules/proc_restricting-custom-repositories-to-a-specific-os-version-or-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Restricting_Custom_Repositories_to_a_Specific_OS_Version_{context}"]
+= Restricting Custom Repositories to a Specific Operating System or Architecture in {Project}
+
+You can configure {Project} to enable custom repositories by default on content hosts with a specific operating system version or architecture.
+For example, you can restrict custom repositories only to {RHEL}{nbsp}9 hosts.
+
+ifdef::satellite[]
+[NOTE]
+ Only restrict architecture and operating system version for custom products.
+{Project} applies these restrictions automatically for {Team} repositories.
```

Comment:
Could `{Team}` make it say something other than "Red Hat"? I don't think it would make sense if it said "Orcharhino repositories" or something like that. "Red Hat repositories" is a special category of repository.

Reply:
Agree to @jeremylenz 
When we are already keeping it for Satellite specific, there is no requirement to use `{Team}`. Also, I have mostly heard of RH repos only when it comes to defining `Repositories`.

Reply:
Thanks! I changed it to Red{nbsp}Hat.